core/util.js

```diff
@@ -366,4 +366,85 @@ Util.Flash = (function () {
     return {version: parseInt(version[0] || 0 + '.' + version[1], 10) || 0, build: parseInt(version[2], 10) || 0};
 }());
 
+
+Util.Localisation = {
+    defaultLanguage: 'en-GB',
+
+    /*
+     * Not all languages have been translated
+     * Some countries prefer a certain language
+     */
+    supportedLanguages: {
+        'en':    'en-GB',
+        'en-GB': 'en-GB',
+        'en-US': 'en-GB',
+        'nl':    'nl-NL',
+        'nl-NL': 'nl-NL',
+        'nl-BE': 'nl-NL',
+        'de':    'de-DE',
+        'de-DE': 'de-DE'
+    },
+
+    // Get language file location
+    getLanguageFileLocation: function () {
+        return '../app/locale/locale-'+Util.Localisation.getLanguageCode()+'.js';
```

Comment:
You mean that we can't expect everyone to have an 'app' or a 'app/locale' directory I guess.. Good point, didn't think of that!